src/app/design/guidelines/back-button/index.html

```diff
@@ -0,0 +1,71 @@
+<stache
+  pageTitle="Back Button"
+  navTitle="Back button"
+  showTableOfContents="true">
+
+  <stache-page-anchor>
+    New page behavior
+  </stache-page-anchor>
+
+  <p>
+    When users navigate to a new context such as a new record, list or section, the action should create an entry in the browser history with a new page title. This entry enables users to select the browser's back button to return to the previous page or view.
```

Comment:
I don't think we need the second "new" before "record."

I might be wrong, but I think "page or view" at the end should just be "view."